src/validations/member.js

```diff
@@ -0,0 +1,28 @@
+const Joi = require('joi');
+
+const validateMembersUpdate = (req, res, next) => {
+  const membersUpdate = Joi.object({
+    firstName: Joi.string().min(3).max(25).required(),
+    lastName: Joi.string().min(3).max(25).required(),
+    dni: Joi.number().min(1000000).max(99999999).required(),
+    phone: Joi.number().required(),
+    email: Joi.string().required(),
+    password: Joi.string().min(8).required(),
+    city: Joi.string().min(8).max(25).required(),
+    birthDay: Joi.date().required(),
```

Comment:
Here you could add a validation to prevent the user from entering a future date.

Reply:
Required validation added..